public/src/app/app.component.html

```diff
@@ -23,13 +23,14 @@
           <span class="navbar-contributions" *ngIf="showContributions">
             <span class="navbar-contribution" *ngFor="let user of status?.contributions.data.repository.contributions.nodes">
               <span *ngIf="user.contributions.pullRequests.nodes.length + user.contributions.reviews.nodes.length + user.contributions.issues.nodes.length > 0">
-                <img src="{{user.avatarUrl}}&size=40">
+                <img class="avatar-40" src="{{user.avatarUrl}}&size=40">
```

Comment:
With the class styling, can the `size=XY` be removed on the img tags?

Reply:
No, that determines the size of bitmap images returned so we don't get 256x256 faces scaled down. It just wasn't working on the default avatars because those are always returned as 420x420 px and scaled -- being very simple geometric shapes the file size doesn't really change.